src/test/java/com/exasol/dbcleaner/ExasolDatabaseCleanerIT.java

```diff
@@ -39,6 +39,22 @@ void testPurgeSchema() throws SQLException {
         assertDoesNotThrow(this::createSchema);
     }
 
+    @Test
+    void testPurgeSchema_userSpecified() throws SQLException {
+        createSchema();
+
+        final CleanRequest request = CleanRequest.builder()
+                .setCleanConnections(false)
+                .setCleanRoles(false)
+                .setCleanUsers(false)
+                .setCleanConnections(false)
+                .setCleanObjects(true)
+                .build();
+
+        CLEANER.cleanDatabase(request);
+        assertDoesNotThrow(this::createSchema);
```

Comment:
This tests only if the object is deleted, not if the other ones are untouched. 50% done. :-)